src/common-models/schedule.model.ts

```diff
@@ -1,6 +1,6 @@
 export interface ScheduleModel {
   UUID?: string;
-  month_number: number;
-  year: number;
+  month_number?: number;
```

Comment:
Zmieniłem typy, bo chciałem, żeby ten switch który dodałem był widoczny w zależności od tego czy jest wczytany grafik czy nie. A sprawdzenie na undefined o wiele bardziej intuicyjnie wyglada w kodzie niz sprawdzenie na to, czy month_number != -1